setup.py

```diff
@@ -61,6 +61,7 @@
 ]
 
 install_requires = [
+    'celery>=3.1.16,<4.0.0',
```

Comment:
Since it's only Python 2.7 we can perhaps do something like:

```python
extras_require = {
    'docs:python_version=="2.7"': [
        # FIXME remove when celery#3993 will be resolved!
        'celery<4.0',
    ]
    'docs': [
        'Sphinx>=1.5.1,<1.6',
    ],
    'tests': tests_require,
}